docs/developer-guide/interactivity.md

```diff
@@ -1,8 +1,72 @@
 # Adding Interactivity
 
-> This article discusses interacting with data (i.e. selecting, or picking objects). Viewport controls (panning, zooming etc) are discussed in [Controllers](/docs/api-reference/map-controller.md).
+## Controlling the camera
 
-## Overview
+In its basic form, deck.gl maintains its own camera state and the keyboard or mouse are used to pan/zoom/tilt layers along with any underlying map or imagery.
+
+To define the initial camera view, set the `initialViewState` property to define the initial camera position. 
+
+For more flexibility you can maintain the view state yourself and pass it in to deck.gl via the `viewState` parameter. This allows you to perform  fly to transitions to a new location, set pitch, zoom or any other property of the viewState.
+
+*Note : Do not combine initialViewState and viewState parameters*
+
+The following example demonstrates the viewState being controlled outside of deck.gl using a setInterval. it also demonstrates the ability to smoothly transition the camera by using the FlyToInterpolator. 
+
+### Using React
+
+```js
+import React from 'react';
+import {DeckGL, ScatterplotLayer} from 'deck.gl';
+import {FlyToInterpolator} from '@deck.gl/core';
+import * as d3 from "d3";
+
+class App extends React.Component {
+
+    state = {
+        zoomedOut: false,
+        viewState: {longitude: -122.45, latitude: 37.78, zoom: 12}
+    }
+
+    componentDidMount() {
+        window.setInterval(() => {
```

Comment:
I don't think this is the best use case for explaining this usage pattern.

New developers are commonly interested in the following use cases:

1. How to get camera interactivity
2. How to reset the camera and/or jump to a location
3. How to apply constraints to the view state (e.g. minZoom, maxPitch, bounding box)

All three can be done in both usage patterns, and the examples should reflect these.